src/openface_node.py

```diff
@@ -236,10 +271,17 @@ def _detect_face_srv(self, req):
         }
 
 if __name__ == '__main__':
-    rospy.init_node('openface')
+    ####################
+    # main creates an instance of OpenfaceROS
+    camera_lr = rospy.get_param('~camera_lr','');
+    if ( not camera_lr ):
+        camera_lr = "_" + camera_lr
+
+    rospy.init_node('openface' + camera_lr)
```

Comment:
In this case, I was trying to have 2 different nodes, one for camera left and another for camera right.  I can remove this since the triangulation function is not fully supported yet.